package-pypi-gowin.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh -ex
+
+# Use venv python to extract Apcula version
+PYTHON=${PYTHON:-apycula-prefix/bin/python}
```

Comment:
This shouldn't be configurable, since we require an exact version match anyway.
```suggestion
PYTHON=apycula-prefix/bin/python
```